vllm/entrypoints/openai/protocol.py

```diff
@@ -29,7 +29,7 @@
 
 from vllm import envs
 from vllm.entrypoints.chat_utils import (ChatCompletionMessageParam,
-                                         random_tool_call_id)
+                                         make_tool_call_id)
```

Comment:
This wouldn't work, given that the default would always be random, so not sure how this would work with kimi here.